demos/connector/neo4j/load-cora-into-neo4j.ipynb

```diff
@@ -306,8 +306,10 @@
    "source": [
```

Comment:
<p>Will this now fail if that environment variable isn't set? It should really default to localhost; could we also use papermill to inject our CI settings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>No,  I've been running them locally as is (against a <code>localhost</code>  Neo4j instance). The <code>get</code>  means it returns <code>None</code>,  and this API accepts <code>host=None</code>  as referring to the default.</p>